firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -2987,6 +2987,20 @@ extension String {
         comment: "Description displayed under the ”Show Link Previews” option")
 }
 
+// MARK: - Open Links in Apps
+extension String {
+    public static let SettingsOpenLinksInAppsTitle = MZLocalizedString(
+        key: "Settings.OpenLinksInApps.Title",
+        tableName: nil,
+        value: "Open Links In Apps",
+        comment: "Title of setting to enable open links in apps when pressing links.")
+    public static let SettingsOpenLinksInAppsStatus = MZLocalizedString(
+        key: "Settings.OpenLinksInApps.Status",
+        tableName: nil,
+        value: "When pressing Links",
+        comment: "Description displayed under the ”Open Links In Apps” option")
```

Comment:
Could we get a copy review on these along with screenshot @dicarobinho

Reply:
or if @afurlan-firefox gives the go ahead for strings then I am okay as well otherwise its best to get these double checked

Reply:
The setting title is from the Android app. Here is how them look.
<img width="477" alt="Screenshot 2024-05-16 at 17 32 37" src="https://github.com/mozilla-mobile/firefox-ios/assets/61138287/0827aac9-aed8-4c20-9882-f3b8c24ed56d">
![iphone](https://github.com/mozilla-mobile/firefox-ios/assets/61138287/f53a4c7e-f361-43e2-b1b5-efa96f00e8ed)